src/internal/views/nunjucks/agreements/delete.njk

```diff
@@ -14,10 +14,14 @@
   <div class="phone--show">
     <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
 
-    <p>Agreement: {{ agreement.agreement.description }}</p>
-    <p>Date signed: {{ agreement.dateSigned | date }}</p>
-    <p>Start date: {{ agreement.dateRange.startDate | date }}</p>
-    <p>End date: {{ agreement.dateRange.endDate | date }}</p>
+    <p>Agreement:
+      {{ agreement.agreement.description }}</p>
```

Comment:
Is the agreement description always populated? If not, might it be worth adding a fallback text? 

Ignore if not.

Reply:
I've added the code as a fallback thanks